__tests__/app.test.js

```diff
@@ -22,14 +22,72 @@ describe('app', () => {
           const { topics } = body;
 
           expect(Array.isArray(topics)).toBe(true);
-          expect(topics.length).toBe(3);
+          expect(topics).toHaveLength(3);
           topics.forEach(topic => {
             expect(topic.hasOwnProperty('slug')).toBe(true);
             expect(topic.hasOwnProperty('description')).toBe(true);
           });
         });
     });
   });
+
+  describe('/api/articles', () => {
+    it('200: GET returns array of article objects', () => {
+      return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          expect(Array.isArray(articles)).toBe(true);
+          expect(articles).toHaveLength(12);
+          articles.forEach(article => {
+            expect(article.hasOwnProperty('title')).toBe(true);
+            expect(article.hasOwnProperty('article_id')).toBe(true);
+            expect(article.hasOwnProperty('topic')).toBe(true);
+          });
+        });
+    });
+    it('200: GET returns each object with comment_count property - total count of comments with this article_id', () => {
+      return request(app)
+        .get('/api/articles')
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+          articles.forEach(article => {
+            expect(article.hasOwnProperty('comment_count')).toBe(true);
+          });
+        });
+    });
+    it('200: GET returns article objects sorted by date, in descending order', () => {
+      return request(app)
+        .get('/api/articles?sort_by=created_at&order=DESC')
+        .expect(200)
+        .then(({ body }) => {
+          const { articles } = body;
+
+          expect(articles).toBeSortedBy('created_at', {
+            descending: true,
+          });
+        });
+    });
+    it('400: GET prevents SQL injection, if user tries to input sort parameter', () => {
```

Comment:
This test description could be clearer, its not _quite_ accurate. 
This test is ensuring that if an invalid sort parameter is provided, your model does not include it as part of your query. 
You're right that we want to prevent SQL injection, but this test is not checking that behaviour eg by attaching an sql query as part of your query. 
We're protecting ourselves by using pg functionality. Which is not something we're explicitly testing ourselves. 
A better description could be `400: GET - responds with error message if an invalid sort_by query is used`